src/index.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
+// import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
+// import reportWebVitals from './reportWebVitals';
```

Comment:
- [ ] Kindly remove all commented-out codes in this file. Leaving such codes make your code look untidy and hard to read. Kindly remove them. 👍